### DISCUSSION TITLE
Sorting Tracker List

### DISCUSSION THREAD

**Discussion by UrinalChills** [c0re100/qBittorrent-Enhanced-Edition#588] [Q&A]
I'm new, and it may be useless...but can tracker list be sorted? this way I can sort by "not working" trackers and delete them.

**Comment by qBittUser** [comment]
Yes, you can sort trackers tab columns.

It's available since v5.0.0 release.

You can also remove a specific tracker from all torrent transfers by right-clicking on a tracker in filters list on the left. If using webUI, then have to wait for v5.1.0 release for this extra feature.

> It may be useless...

Depends.

Too many trackers or potentially a duplicate tracker can overload network and cause delays announcing to some trackers, but too few trackers can also lower chance of finding peers.

Sometimes trackers are incorrectly added without a empty line between, have a typo, users might be even using wrong advanced settings, there can just be a temporary connection issue, used DNS server might not allow connecting to some, some torrents might not be registered or your torrenting client version just isn't whitelisted.

So you sometimes need patience to see which trackers are unnecessary to keep.
